### ISSUE TITLE
Latest docker image still has mintapi 2.5 installed

### ISSUE THREAD

**Issue by sfitzgerald96** [mintapi/mintapi#528]
**Describe the bug**
Downloading the latest docker image still has mintapi 2.5 installed.

**What version of MintAPI are you using?**
I'm running the MintAPI on the latest docker image with sha256:4a6b2114f1388586869fd879d81ece9fe2467c64cfac45a44f4f8fbee8fa8cc2

**What command[s] did you run / steps to reproduce?**
`docker run --rm ghcr.io/mintapi/mintapi@sha256:4a6b2114f1388586869fd879d81ece9fe2467c64cfac45a44f4f8fbee8fa8cc2 pip list -v | grep mintapi`

**Stacktrace/error received:**
When using the mintapi cli, I'm getting the same stacktrace errors as listed in #515 

**What did you expect to happen?**
I'd expect the version to be 2.6

**What actually happened?**
The version was 2.5

**Comment by burkematthew** [comment]
@sfitzgerald96 is the only reason you think it's not 2.5 because you are receiving the error in the linked issue? Can you do a check of the version installed in the container?

**Comment by sfitzgerald96** [comment]
@burkematthew Thanks for the quick reply! Yeah, I checked the version by running `pip list -v` in the container and it comes back as version 2.5

I ran `docker run --rm ghcr.io/mintapi/mintapi:latest pip list -v` and it shows mintapi as version 2.5. I wanted to make sure it wasn't my local machine caching the previous `latest` tag, which is why I ran it with the sha instead as seen in the command listed above.

I did a workaround for my use case by extending the docker image and running `pip install mintapi --upgrade`. The output showed it upgraded to 2.6 and then my script started working, which is what made me think version 2.6 didn't quite make it into this current docker image.
